java2cpp.l

```diff
@@ -38,6 +38,8 @@
 "void"                {return VOID;}
 "System.out.println"   {return PRINTLN;}
 "System.out.print"	    {return PRINT;}
+"Scanner"                {return SCANNER;}
+"System.in"              {return SYS.IN}
```

Comment:
Rename ```SYS.IN``` to ```SYS_IN```
> This is needed because Lex interprets ```var1.var2``` as a struct or object. So the dot cannot be used in a variable name.
Yous must also add a semicolon ; after it.

It will look like this:
```c
"System.in"              {return SYS_IN;}
```